roles/ezid/tasks/main.yml

```diff
@@ -4,3 +4,19 @@
   become: yes
   template: src=ezid.yml.j2 dest={{ config_path }}/ezid.yml \
             owner={{ deploy_user }} group={{ deploy_group }} backup=no
+
+# encoding the YAML replacement in base64 since colons in YAML strings cause problems
+- name: write Ezid user to secrets file
+  become: yes
+  shell: sed -i "$(echo 'L1woZGVmYXVsdDpcIFwmZGVmYXVsdFwpL2NkZWZhdWx0OlwgXCZkZWZhdWx0XG5cIFwgZXppZF91c2VyOlwgCg==' | base64 -d)$(echo '{{ ezid_user }}')" \
```

Comment:
The decoded string is `/\(default:\ \&default\)/cdefault:\ \&default\n\ \ ezid_user:\` and so on for the other tasks.